samples/run.ts

```diff
@@ -17,7 +20,57 @@ if (selection) {
     samples = [selection];
 }
 
+async function createProject(projectId: string, coreApiObject: coreApi.ICoreApi) : Promise<void> {
+    console.log('Cleaning up from last run');
+    await destroyProject(projectId, coreApiObject);
+
+    const projectToCreate: coreInterfaces.TeamProject = {name: projectId,
+                                                         description: 'sample project created automatically by azure-devops-node-api.',
+                                                         visibility: 1,
+                                                         capabilities: {versioncontrol: {sourceControlType: 'Git'}, 
+                                                                        processTemplate: {templateTypeId: '6b724908-ef14-45cf-84f8-768b5384da45'}},
+                                                         _links: null,
+                                                         defaultTeam: null,
+                                                         abbreviation: null,
+                                                         id: null,
+                                                         revision: null,
+                                                         state: null,
+                                                         url: null};
+    await coreApiObject.queueCreateProject(projectToCreate);
+
+    //Poll until project exists
+    let projIfExists: coreInterfaces.TeamProject = null;
+    console.log('Waiting for project to spin up');
+    while (!projIfExists) {
+        projIfExists = await coreApiObject.getProject(projectId);
+    }
+}
+
+async function destroyProject(projectId: string, coreApiObject: coreApi.ICoreApi){
```

Comment:
deleteProject. Same comment about core api. Also don't need to call it "object".